public/remote.html

```diff
@@ -0,0 +1,94 @@
+
+    const chartHTML = document.getElementById('salarychart')
+
+    const data = salaryJSON.map(salary => salary.salary)
+
+    const labels = data.filter((value, index, self) => self.indexOf(value) === index)
+
+		const remoteTrue = []
+    const remoteFalse = []
+
+    for (let i = 0; i < salaryJSON.length; i++) {
+			if (salaryJSON[i].remote == true) {
+      	remoteTrue.push(salaryJSON[i].salary)
+      } else {
+      	remoteFalse.push(salaryJSON[i].salary)
+      }
+		}
+
+    const frequency = []
+
+    for (let i = 0; i < labels.length; i++) {
+      const num = labels[i]
+      const count = remoteFalse.reduce(function(n, val) {
+        return n + (val === num);
+      }, 0);
+      frequency.push(count)
+    }
+
+    const remoteFrequency = []
+
+    for (let i = 0; i < labels.length; i++) {
+      const num = labels[i]
+      const count = remoteTrue.reduce(function(n, val) {
```

Comment:
If you replaced `remoteTrue` here and `remoteFalse` above with

```
salaryJSON.filter(salary => salary.remote).reduce.......
```

you'd cut the entire `remoteTrue`/`remoteFalse` loop above, which saves 9 lines of code.